.github/workflows/windows.yml

```diff
@@ -15,4 +15,24 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: .\scripts\install_zig.bat
-      - run: zig\zig.exe build c_sample -Drelease-safe
\ No newline at end of file
+      - run: zig\zig.exe build c_sample -Drelease-safe
```

Comment:
I didn't actually make a change to this line. In any future PR we could make that change yes but it's unrelated to my changes.

Reply:
Ok I made this change for this PR but it doesn't actually matter since scripts\build.bat is a dumb wrapper. There's no point in making this change everywhere IMO.